templates/zerver/help/configure-email-notifications.md

```diff
@@ -0,0 +1,32 @@
+# Configure email notifications
+
+Zulip allows you to configure your email notifications preferences. You have a
+choice to receive or not receive email notification when you're offline.
+
+You can easily change your email notification settings by following these steps.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right
+corner of the right sidebar.
+
+2. Select **Settings** from the dropdown menu that appears.<br>
+
+  ![Settings](/static/images/help/cog_dropdown_settings.png)
+
+3. Upon clicking **Settings**, your view will be replaced with the **Settings**
+page. Scroll down to the **Notifications** section (after the **Display
+Settings** section).
+
+4. Under the **Private messages and @-mentions** section of **Notifications**
+you should find a check box with the label **Email notifications when
+offline**. When this checkbox is selected, you will receive email notifications
+whenever you're offline.
+
+  ![Settings](/static/images/help/notifications-settings.png)
+
+5. Under the **Other notifications *I want*** section of **Notifications** you
```

Comment:
Preview this file and check this line. I don't think its previewing the view you wanted it be.